src/opera/test/data/test_dswx_ni_config.yaml

```diff
@@ -20,10 +20,15 @@ RunConfig:
       PrimaryExecutable:
         ProductIdentifier: DSWX_NI
         ProductVersion: "1.0"
-        ProgramPath: /bin/echo
+        ProgramPath: mkdir
         ProgramOptions:
-        - hello world > dswx_ni_pge_test/output_dir/OPERA_L3_DSWx-NI_T18MVA_20200702T231843Z_20230317T190549Z_v0.1_B01_WTR.tif;
-        - /bin/echo DSWx-NI invoked with RunConfig
+          - '-p dswx_ni_pge_test/output_dir;'
```

Comment:
Maybe add a comment or 2.  Looks like your making a bunch of 1M test files filled with random data?

Reply:
I placed this in the wrong place.  I'm talking about the dd command below this line